apps/api/src/form/swagger.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  CurrentEmploymentTypes,
+  DeploymentDurations,
+  HealthAuthorities,
+  RegistrationStatus,
+  StreamTypes,
+} from '@ehpr/common';
+import { SchemaObject } from '@nestjs/swagger/dist/interfaces/open-api-spec.interface';
+
+const primaryInformationSchema = {
```

Comment:
I saw `ApiProperty` decorator with DTO object to specify schema, to keep single source of truth for schema. Any specific reason of not using that approach ?

Reply:
I agree, I think I'd prefer adding the swagger tags into the common package to reduce duplication

Reply:
I actually spent a good deal of time using nestjs decorators for Swagger. However, it's heavily coupled with the framework to the point I had a lot of dependencies. Also, putting them in devDependencies did not work at all even if I had all the necessary dependencies.
Also, a common package should only include functionality used by both app and API. I would argue against putting swagger schema since it is not being used in the FE.

Reply:
If it'll be too heavyweight to include Swagger in common, let's leave it here

Can we make properties extend types for each form step to make sure they don't go out of sync?

Reply:
@arranfw not sure I fully understand.

Reply:
Was thinking of adding types to the properties field so changes in the DTO would throw TS errors here if it isn't updated

Reply:
Lets revisit this later.